daemon/src/model.rs

```diff
@@ -105,3 +105,6 @@ pub struct WalletInfo {
     pub address: Address,
     pub last_updated_at: SystemTime,
 }
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq, Hash)]
+pub struct OracleEventId(pub String);
```

Comment:
❓ Would it make sense to put this type in `oracle` and name it `Id`?

Reply:
This type is now used in various modules - it is also serialized/deserialized as part of the `Order`. We could be cleaner here, and map it to a specific type for serialization in `to_sse_event`. I admit I was too lazy for that. But I would keep it in `model` and treat it similar to e.g. `Usd` which is also ussed throughout the codebase but directly serialized for the UI.

Reply:
Would be nice to parse this one into its semantic parts at some point :)